functions.py

```diff
@@ -635,40 +636,36 @@ async def get_location_overlaps(target_uri, output_featuretype_uri, include_area
     overlaps_sparql = """\
 PREFIX geo: <http://www.opengis.net/ont/geosparql#>
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX ipo: <http://purl.org/dc/terms/isPartOf> 
+PREFIX dcterms: <http://purl.org/dc/terms/>
+# PREFIX ipo: <http://purl.org/dc/terms/isPartOf> 
 PREFIX geox: <http://linked.data.gov.au/def/geox#>
-PREFIX qb4st: <http://www.w3.org/ns/qb4st/>
+# PREFIX qb4st: <http://www.w3.org/ns/qb4st/>
 PREFIX epsg: <http://www.opengis.net/def/crs/EPSG/0/>
 PREFIX dt: <http://linked.data.gov.au/def/datatype/>
 SELECT <SELECTS>
 WHERE {
+    ?p rdfs:subPropertyOf* geo:sfOverlaps .
```

Comment:
this would work, except we haven't loaded the geox ontology in the cache yet. 

See http://3.104.29.122/sparql?name=&infer=true&sameAs=true&query=PREFIX+geox%3A+%3Chttp%3A%2F%2Flinked.data.gov.au%2Fdef%2Fgeox%23%3E%0APREFIX+rdfs%3A+%3Chttp%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23%3E%0Aselect+*%0Awhere+%7B%0A++++geox%3AtransitiveSfOverlap++++rdfs%3AsubPropertyOf+%3Fs+%3B++++%0A%7D

question is _should_ the cache load the geox ontology also?